src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -185,6 +188,15 @@ If a user has access to several organizations that use New Relic (for example, i
 
 If you're migrating users in chunks and not all at once, you can go through the migration workflow several times with different groups of users. You can only click **Finish Setup** when all users in the organization are migrated.
 
+## Troubleshooting [#troubleshooting]
+
+Some common problems after migration: 
+
+* If you have admin-level roles assigned but get an error message when trying to access New Relic platform features, it may be because you've been assigned organization-scoped roles (**Organization manager** and/or **Authentication domain manager**) but not any account-scoped roles. To access New Relic features in a specific account, you'll need at least one account-scoped role (for example, **All product admin** or a custom role). 
```

Comment:
@zuluecho9, are you saying these account-scoped roles don't come over during the migration?